src/storj.h

```diff
@@ -48,80 +46,66 @@ extern "C" {
 #define STORJ_TRANSFER_OK 0
 #define STORJ_TRANSFER_CANCELED 1
 
-// Bridge related errors 1000 to 1999
-#define STORJ_BRIDGE_REQUEST_ERROR 1000
-#define STORJ_BRIDGE_AUTH_ERROR 1001
-#define STORJ_BRIDGE_TOKEN_ERROR 1002
-#define STORJ_BRIDGE_TIMEOUT_ERROR 1003
-#define STORJ_BRIDGE_INTERNAL_ERROR 1004
-#define STORJ_BRIDGE_RATE_ERROR 1005
-#define STORJ_BRIDGE_BUCKET_NOTFOUND_ERROR 1006
-#define STORJ_BRIDGE_FILE_NOTFOUND_ERROR 1007
-#define STORJ_BRIDGE_JSON_ERROR 1008
-#define STORJ_BRIDGE_FRAME_ERROR 1009
-#define STORJ_BRIDGE_POINTER_ERROR 1010
-#define STORJ_BRIDGE_REPOINTER_ERROR 1011
-#define STORJ_BRIDGE_FILEINFO_ERROR 1012
-#define STORJ_BRIDGE_BUCKET_FILE_EXISTS 1013
-#define STORJ_BRIDGE_OFFER_ERROR 1014
-
-// Farmer related errors 2000 to 2999
-#define STORJ_FARMER_REQUEST_ERROR 2000
-#define STORJ_FARMER_TIMEOUT_ERROR 2001
-#define STORJ_FARMER_AUTH_ERROR 2002
-#define STORJ_FARMER_EXHAUSTED_ERROR 2003
-#define STORJ_FARMER_INTEGRITY_ERROR 2004
+// Libuplink error (i.e. check STORJ_LAST_ERROR)
+#define STORJ_LIBUPLINK_ERROR 1000
```

Comment:
This looks to a new error constant. Let's use a number that was not used before.